### PR TITLE
CCITCARBON-237 Docs restructure for github pages

### PR DESCRIPTION
**Pull request by shay6**
* Fix tables for GH pages
* Update docs structure

### CODE REVIEW

File: docs/user.md [hunk 1]
```diff
@@ -662,4 +667,4 @@ since Teflo does not have similar functionality. Where we suggest users should s
 away from Linchpin hooks is if any `postup` hooks have been configured. That is where thos scripts/playbooks
 can be moved over and configured in Teflo's 
 [Orchestrate](https://docs.engineering.redhat.com/display/CentralCI/Orchestrate) section. It can help
```
Comment (rujutashinde) [inline docs/user.md]: Need to change this link to the correct one from readthedocs. This is pointing to the older docs on confluence
https://teflo.readthedocs.io/en/latest/users/definitions/orchestrate.html

**Review by rujutashinde — COMMENTED** [review]
one change in link

File: docs/user.md (hunk 1 above)
Comment (shay6) [reply docs/user.md]: done

**Review by rujutashinde — APPROVED** [review]
LGTM

**Pull request merged**